WorkflowFunctionAdminBundle/Transformer/WorkflowFunctionCollectionTransformer.php

```diff
@@ -20,7 +19,7 @@ class WorkflowFunctionCollectionTransformer extends AbstractSecurityCheckerAware
      */
     public function transform($mixed)
     {
-        $facade = new WorkflowFunctionCollectionFacade();
+        $facade = $this->newFacade();
```

Comment:
@Thiblef : Can you throw an exception ``if (!$facade instanceof FacadeInterface)``